pvlib/shading.py

```diff
@@ -342,3 +342,144 @@ def projected_solar_zenith_angle(solar_zenith, solar_azimuth,
+    cross_axis_slope=0,
+    shading_tracker_tilt=None,
+):
+    r"""
+    Shaded fraction in the vertical dimension of tilted rows.
+
+    If ``shading_tracker_tilt`` isn't provided, assumes both the shaded
+    row and the one blocking the direct beam
+    share the same tilt and azimuth values.
+
+    If only the GCR is known, feed GCR into ``trackers_vertical_length`` and
+    specify ``trackers_row_pitch=1``.
+
+    .. versionadded:: 0.10.5
+
+    Parameters
+    ----------
+    solar_zenith : numeric
+        Solar position zenith, in degrees.
+    solar_azimuth : numeric
+        Solar position azimuth, in degrees.
+    trackers_axis_azimuth : numeric
+        In degrees. North=0º, South=180º, East=90º, West=270º.
+    shaded_tracker_tilt : numeric
+        Tilt of the tracker receiving the shade in degrees.
+    collector_width : numeric
+        Vertical length of a tilted tracker. The returned ``shaded fraction``
+        is the ratio of the shadow over this value.
+    row_pitch : numeric
+        Axis-to-axis horizontal spacing of the trackers.
+    surface_to_axis_offset : numeric, default 0
+        Distance between the rotating axis and the collector surface.
+    cross_axis_slope : numeric, default 0
+        Angle of the plane containing the rows' axes from
+        horizontal. In degrees.
+    shading_tracker_tilt : numeric, optional
+        Tilt of the tracker casting the shadow.
+
+    Returns
+    -------
+    shaded_fraction : numeric
+        The fraction of the collector width shaded by an adjacent row. A
+        value of 1 is completely shaded and zero is no shade.
+
+    Notes
+    -----
+    All length parameters must have the same units to produce a reasonable
+    result.
+
+    Parameters are defined as follow:
+
+    .. figure:: ../../_images/Anderson_Jensen_2024_Fig3.png
+        :alt: Diagram showing the two trackers and the parameters of the model.
+
+        Figure 3 of [1]_. See correspondence between this nomenclature and the
+        function parameters in the table below.
+
+    +------------------+----------------------------+---------------------+
+    | Symbol           | Parameter                  | Units               |
+    +==================+============================+=====================+
+    | :math:`\theta_1` | ``shading_tracker_tilt``   |                     |
+    +------------------+----------------------------+                     |
+    | :math:`\theta_2` | ``shaded_tracker_tilt``    | Degrees             |
+    +------------------+----------------------------+ :math:`^{\circ}`    |
+    | :math:`\beta_c`  | ``cross_axis_slope``       |                     |
+    +------------------+----------------------------+---------------------+
+    | :math:`p`        | ``row_pitch``              | Any consistent      |
+    +------------------+----------------------------+ length unit across  |
+    | :math:`l`        | ``collector_width``        | all these           |
```

Comment:
```suggestion
    | :math:`\ell`        | ``collector_width``        | all these           |
```

This is the latex command for the same L character used in the diagram.  Hopefully it displays correctly with mathjax :)

Reply:
Yay!! It does! https://pvlib-python--1962.org.readthedocs.build/en/1962/reference/generated/pvlib.shading.shaded_fraction1d.html